### PR TITLE
US69524: Adjust spacing around the side Two Panel Selector search bar.

### PR DESCRIPTION
**Pull request by njostonehouse**

### CODE REVIEW

File: src/page-two-panel-selector.scss [hunk 1]
```diff
@@ -44,11 +44,9 @@
 }
 
 .d2l-twopanelselector-side .d2l-page-search {
-	margin-left: 20px;
-	[dir='rtl'] & {
+	margin-bottom: 20px;
 		margin-left: 0;
-		margin-right: 20px;
-	}
+	padding: 0 30px;
```
Comment (dlockhart) [inline src/page-two-panel-selector.scss]: Is the intent `30px` or `1.5rem`? (the search inputs internal padding is rem-based)

Comment (njostonehouse) [inline src/page-two-panel-selector.scss]: It should be constant. This is the padding around the search field, to make sure it's the same width as the side panel below it (also will have 30px padding).

**Comment by njostonehouse** [comment]
This PR is associated with LE PR 4461.

File: src/page-two-panel-selector.scss [hunk 2]
```diff
@@ -44,11 +44,9 @@
 }
 
 .d2l-twopanelselector-side .d2l-page-search {
-	margin-left: 20px;
-	[dir='rtl'] & {
-		margin-left: 0;
-		margin-right: 20px;
-	}
+	margin-bottom: 20px;
+	margin-left: 0;
```
Comment (dbatiste) [inline src/page-two-panel-selector.scss]: Is this overriding some left margin applied elsewhere?  If so, is there a similar RTL override needed?

Comment (dlockhart) [inline src/page-two-panel-selector.scss]: Agree with Dave's comment. If it's not overriding something, then the rule can just be excluded (things get 0 margin by default).

Comment (njostonehouse) [inline src/page-two-panel-selector.scss]: Yep, I missed having to override PageSearch.RTL.css. Thanks guys.

**Pull request merged**